.github/workflows/publish_wiki.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish Wiki
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "docs/wiki/**"
+  workflow_dispatch:
+
+concurrency:
+  group: publish-wiki
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
Is there a reason to use v1 instead of [v4](https://github.com/actions/checkout?tab=readme-ov-file#usage)?

Reply:
it should probably be bumped, but dependabot will update it anyway